lib/ruby_lsp/ruby-lsp-cell/code_lens.rb

```diff
@@ -64,6 +64,20 @@ def on_class_node_enter(node)
 
         @in_cell_class = true
         add_default_goto_code_lens(node)
+
+        erb_views = find_uri_views
```

Comment:
<img width="388" alt="image" src="https://github.com/user-attachments/assets/cca13534-36a8-4f9f-b019-b39a9447d861" />

no mostraría el "Search view" si es un solo .erb y justamente es el default

Reply:
No me decidí tampoco, estaba buena, pero lo más agradable sería un "go to definition" en las vistas mismas. Efectivamente pasa harto que termina haciendo lo mismo que el go to show al ser una pura vista, y que esté en algunos casos y en otros no puede confundir. Lanzemos sin esto y nos vamos por lo simple y rápido primero.